app/components/institution-page/campus-program-application-modal/verify-email-step-container.ts

```diff
@@ -1,15 +1,22 @@
 import Component from '@glimmer/component';
 import type InstitutionMembershipGrantApplicationModel from 'codecrafters-frontend/models/institution-membership-grant-application';
+import { action } from '@ember/object';
 
 interface Signature {
   Element: HTMLDivElement;
 
   Args: {
     application: InstitutionMembershipGrantApplicationModel;
+    onGoBackToEnterEmail: () => void;
   };
 }
 
-export default class VerifyEmailStepContainer extends Component<Signature> {}
+export default class VerifyEmailStepContainer extends Component<Signature> {
+  @action
+  handleChangeEmailButtonClick() {
+    this.args.onGoBackToEnterEmail();
```

Comment:
For simple cases like this just pass the arg directly in the template. `<button {{on "click" @onWhatever}}`.